src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,932 @@
+
+- [Constraint]({% link {{ page.version.version }}/constraints.md %}) violations will abort transactions at all isolation levels.
+
+- In rare cases under `READ COMMITTED` isolation, a [`RETRY_WRITE_TOO_OLD`]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}#retry_write_too_old) or [`ReadWithinUncertaintyIntervalError`]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}#readwithinuncertaintyintervalerror) error will be returned to the client if a statement has already begun streaming a partial result set back to the client and cannot retry transparently. By default, the result set is buffered up to 16 KiB before overflowing and being streamed to the client. You can configure the result buffer size using the [`sql.defaults.results_buffer.size`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-defaults-results-buffer-size) cluster setting or the [`results_buffer_size`]({% link {{ page.version.version }}/session-variables.md %}) session variable.
+
+### Concurrency anomalies
+
+Statements in concurrent `READ COMMITTED` transactions can interleave with each other. This can create concurrency anomalies that are not permitted under `SERIALIZABLE` isolation, which places concurrent transactions into a [serializable ordering]({% link {{ page.version.version }}/demo-serializable.md %}).
+
+{{site.data.alerts.callout_success}}
+The behaviors described in this section assume the use of non-locking reads. You can prevent concurrency anomalies through the selective use of [locking reads](#locking-reads), which can also increase latency due to [lock contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention).
+{{site.data.alerts.end}}
+
+#### Write skew anomaly
+
+The following sequence of operations is possible under `READ COMMITTED` isolation: 
+
+1. Transaction `A` reads row `R` in table `T` at timestamp `1`.
+1. Transaction `B` writes to row `R` in table `T` and commits at timestamp `2`.
+1. Transaction `A` writes to another row in table `T` and commits at timestamp `3`.
+
+The value of `R` has changed while transaction `A` is open. However, `A` can still write and commit instead of aborting, since `READ COMMITTED` transactions do not require serializability. This does not inherently create an anomaly, but is the basis of potential *write skew anomalies* where two concurrent transactions each read values that the other subsequently updates.
+
+{{site.data.alerts.callout_info}}
+For details on why this is allowed, see [Read refreshing]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#write-skew-tolerance).
+{{site.data.alerts.end}}
+
+##### Example: Write skew anomaly
+
+For an example of how a write skew anomaly can occur, see [Demonstrate interleaved statements in `READ COMMITTED` transactions](#demonstrate-interleaved-statements-in-read-committed-transactions).
+
+#### Lost updates
```

Comment:
`s/Lost update/Lost update anomaly/`

Here and throughout, to avoid confusion with possible other meanings of "lost updates".